llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -4181,13 +4181,20 @@ bool SIInstrInfo::isInlineConstant(const MachineOperand &MO,
   case AMDGPU::OPERAND_REG_INLINE_C_V2INT16:
   case AMDGPU::OPERAND_REG_INLINE_AC_V2INT16:
     return AMDGPU::isInlinableLiteralV2I16(Imm);
+  case AMDGPU::OPERAND_REG_IMM_V2BF16:
```

Comment:
It does not seem isInlinableLiteralV2F16() can handle bf16.

Reply:
Yeah, I made some mistakes here. Will take care of them.